app/assets/styles/styles.css

```diff
@@ -33,194 +31,120 @@ p {
   line-height: 1.55;
 }
 
-.newsletter-date {
-  color: #fdfd56;
-  font-size: 0.8rem;
+.logo__btn {
+  display: flex;
+  justify-content: center;
+  margin-top: 0.7em;
+  margin-bottom: 0.7em;
 }
 
-.language-menu {
-  width: 100%;
-  font-size: 1.15rem;
-  font-family: 'Trajan Pro Regular', 'Times New Roman', serif;
-  text-align: right;
-  background: #1a1451;
-  margin-bottom: 0.625em;
-  padding: 0.5em 1em;
+@media only screen and (min-width: 400px) {
+  .logo__btn {
+    display: block;
+    position: absolute;
+    top: 0;
+    margin-top: 0;
+    margin-left: 1.5em;
+  }
 }
 
-.language-menu--item {
-  color: #fff;
-  padding: 0.475em;
+.menu {
+  display: flex;
+  justify-content: space-between;
+  font-size: 0.95rem;
+  font-family: 'Trajan Pro Bold', 'Times New Roman', serif;
+  background: #1a1451;
+  padding: 0.5em 1.5em;
```

Comment:
Can we move these styles to the parent element `nav`? Conceptually, whenever possible, I find it easier to apply background colors and padding to the parent element. Then the children are placed inside it. Sometimes we can't put padding on the parent and it needs to go on the children, for example if we want a child element to have a background color and fill the full height of the parent. 

The other advantage is that by applying the padding to the parent element, you won't need to apply a left margin to the logo and it will be positioned with the same amount of horizontal space. 

Also, by decoupling the padding from the menu you can add the margin-left to the menu instead of the name.

Reply:
I got it

![new-header-design-newsletter-nav-large](https://user-images.githubusercontent.com/29107510/82350436-5a426b80-99c9-11ea-9382-2badfc5ebd14.png)

But, the logo is inside the nav. For mobile this is how it looks

![new-header-design-newsletter-bg-nav](https://user-images.githubusercontent.com/29107510/82350171-0041a600-99c9-11ea-8eb2-072e10b1532d.png)

It can be fixed if we move out the logo from the nav. Should I do it?

![new-header-design-newsletter-logo](https://user-images.githubusercontent.com/29107510/82353828-f3738100-99cd-11ea-8fde-cfdf26ce2daf.png)